client/src/main/java/io/split/client/HttpSplitChangeFetcher.java

```diff
@@ -38,16 +35,16 @@ public final class HttpSplitChangeFetcher implements SplitChangeFetcher {
     private static final String HEADER_CACHE_CONTROL_NAME = "Cache-Control";
     private static final String HEADER_CACHE_CONTROL_VALUE = "no-cache";
```

Comment:
please, remove both strings, not using them anymore.

Reply:
will do